OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -301,9 +301,9 @@ protected function generateNonce()
     /**
      * Configure the option resolver
      *
-     * @param OptionsResolverInterface $resolver
+     * @param OptionsResolver $resolver
      */
-    protected function configureOptions(OptionsResolverInterface $resolver)
+    protected function configureOptions(OptionsResolver $resolver)
```

Comment:
@Soullivaneuh it is a BC break for anyone implementing their own resource owners. And resource owners are an extension point of this bundle, not an internal class.

Regarding the removal, there is no need to create a new branch with it removed yet. BC layers don't hurt much. It is not worth maintaining multiple parallel versions of the bundle just because of a BC break removing a small BC layer.